keyboards/dichotomy/config.h

```diff
@@ -41,6 +41,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define ONESHOT_TIMEOUT 500
 
+
+#define TAPPING_TERM 200
+#define TAPPING_TOGGLE 2
```

Comment:
This should really go in your keymap level config.h.

Reply:
Thanks for the tip! I did not know you could have a config on keymaps itself.
I updated the PR